src/script/page/LeftSidebar/panels/Conversations/ConversationTabs/TeamCreation/TeamCreation.styles.ts

```diff
@@ -19,6 +19,8 @@
 
 import {CSSObject} from '@emotion/react';
 
+import {media} from '@wireapp/react-ui-kit';
```

Comment:
and here it is media? :)

Reply:
https://github.com/wireapp/wire-webapp/pull/18254#discussion_r1836515517
Same as above.